packages/core/src/components/button/buttonGroup.tsx

```diff
@@ -15,6 +15,8 @@ export interface IButtonGroupProps extends IProps, React.HTMLProps<HTMLDivElemen
      * `align="left"` will left-align button text and push `rightIcon` to right side.
      * `align="right"` right-aligns text and pushes `icon` to left side.
      * This prop only has an effect if buttons are wider than their default widths.
+     *
+     * The `Alignment` enum provides constants for allowed values.
```

Comment:
same

Reply:
string literals are not assignable to enum types (we learned this with string enums #1921)